### PR TITLE
Indexing & QC2 (final edits before going to print)

### PR DESCRIPTION
**Pull request by hjwp**

### CODE REVIEW

File: appendix_csvs.asciidoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 == Swapping Out the Infrastructure: pass:[<span class="keep-together">Do Everything with CSVs</span>]
 
 This appendix is intended as a little illustration of the benefits of the
-Repository, Unit of Work, and Service Layer patterns. It's intended to
+Repository, Unit of Work, and Service Layer patterns.((("CSVs, doing everyting with", id="ix_CSV"))) It's intended to
```
Comment (xtaje) [inline appendix_csvs.asciidoc]:
```suggestion
Repository, Unit of Work, and Service Layer patterns.((("CSVs, doing everything with", id="ix_CSV"))) It's intended to
```

File: appendix_csvs.asciidoc [hunk 2]
```diff
@@ -289,7 +289,7 @@ def main(folder):
 ====
 
 
-Ta-da! _Now are y'all impressed or what_?
+Ta-da! _Now are y'all impressed or what_?((("CSVs, doing everyting with", startref="ix_CSV")))
```
Comment (xtaje) [inline appendix_csvs.asciidoc]:
```suggestion
Ta-da! _Now are y'all impressed or what_?((("CSVs, doing everything with", startref="ix_CSV")))
```

File: chapter_03_abstractions.asciidoc [hunk 3]
```diff
@@ -663,21 +663,21 @@ act as a record of our design choices, and serve to explain the system to us
...
 Tests that use too many mocks get overwhelmed with setup code that hides the
-story we care about.
+story we care about.((("mocking", "overmocked tests, pitfalls of")))
 
 Steve Freeman has a great example of overmocked tests in his talk
 https://oreil.ly/jAmtr["Test-Driven Development: That's Not What We Meant"].
 You should also check out this https://oreil.ly/s3e05[PyCon talk on Mocking Pitfalls]
 by our esteemed tech reviewer, Ed Jung, which also addresses mocking and its
-alternatives.
+alternatives.((("&quot;Test-Driven Development: That&#x27;s Not What We Meant&quot;", primary-sortas="Test-Driven Development")))((("Freeman, Steve")))((("PyCon talk on Mocking Pitfalls")))((("Jung, Ed")))
```
Comment (hjwp) [inline chapter_03_abstractions.asciidoc]: @xtaje :wink:

File: appendix_django.asciidoc [hunk 4]
```diff
@@ -104,8 +104,12 @@ but it is still made up of familiar-looking Django code:
...
     sku = "PONY-STATUE"
     d_line = django_models.OrderLine.objects.create(orderid="order1", sku=sku, qty=12)
-    d_b1 = django_models.Batch.objects.create(reference="batch1", sku=sku, qty=100, eta=None)
-    d_b2 = django_models.Batch.objects.create(reference="batch2", sku=sku, qty=100, eta=None)
+    d_b1 = django_models.Batch.objects.create(
+    reference="batch1", sku=sku, qty=100, eta=None
+)
+d_b2 = django_models.Batch.objects.create(
+    reference="batch2", sku=sku, qty=100, eta=None
+)
```
Comment (xtaje) [inline appendix_django.asciidoc]: Looks like the indent got mangled here.

**Pull request merged**